library/Freckle/App/Test.hs

```diff
@@ -15,6 +15,7 @@ import Freckle.App.Prelude
 
 import Control.Monad.Base
 import Control.Monad.Catch
+import qualified Control.Monad.Fail as Fail
```

Comment:
This is the recommend approach, compability-wise. The import is necessary on older GHCs, and redundant on newer ones. Doing it qualified allows it to be present for either.